### PR TITLE
Add io.github.baldeuniversel.am-okay

### PR DESCRIPTION
**Pull request by baldeuniversel**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
submissions must be made against the `new-pr` branch.

**Comment by hfiguiere** [comment]
instead you delete all the history of the previous reviews for no reason.

**Comment by baldeuniversel** [comment]
> instead you delete all the history of the previous reviews for no reason.

Sorry, that wasn't my intention, I just don't understand you.  
Please, paraphrase your phase `submissions must be made against the new-pr branch`. I need to know what to do exactly.

Thanks.

File: io.github.baldeuniversel.am-okay.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+finish-args:
+    - --filesystem=host
+
+modules:
+    - name: am-okay-package
+      buildsystem: simple      
+      sources:
+          - type: git 
+            url: https://github.com/baldeuniversel/am-okay.git 
+            branch: new-pr
```
Comment (hfiguiere) [inline io.github.baldeuniversel.am-okay.yml:17]: yet another time: you can not have a git branch here. This is not an optional suggestion.

**Comment by baldeuniversel** [comment]
> image

Is that my **.yml* file has to be in the `master` branch (I'm confused) ?

**Pull request closed without merging**